changelog.md

```diff
@@ -1,4 +1,8 @@
 # Change Log
+## 3.1.4
+**Fixes**
+ * Instead of ForbiddenAccess for denied Include, filter to empty set.
```

Comment:
Missing note about bailing when unable to parse permission strings

Reply:
Added.